src/TokenizerFwf.cpp

```diff
@@ -6,15 +6,26 @@ using namespace Rcpp;
 #include "TokenizerFwf.h"
 #include "utils.h"
 
-std::vector<bool> emptyCols_(SourceIterator begin, SourceIterator end, size_t n = 100) {
+std::vector<bool> emptyCols_(SourceIterator begin, SourceIterator end, size_t n = 100, std::string comment = "") {
 
   std::vector<bool> is_white;
 
   size_t row = 0, col = 0;
+  bool hasComment = comment.length() > 0;
   for (SourceIterator cur = begin; cur != end; ++cur) {
     if (row > n)
       break;
 
+    if (col == 0 && hasComment) {
+      boost::iterator_range<const char*> haystack(cur, end);
```

Comment:
You can (I am doing so here with the comment parameter), but we don't have a string for the source in this function (unless I am missing something).